airbyte-integrations/connectors/source-open-exchange-rates/integration_tests/abnormal_state.json

```
@@ -0,0 +1,5 @@
+{
+  "open_exchange_rates": {
+    "date": "2121-05-18"
```

We'll have to put a timestamp here as well

```suggestion
    "timestamp": 2052084644
```